ingestion/src/metadata/ingestion/source/database/redshift/metadata.py

```diff
@@ -168,6 +173,31 @@ def create(
         )
         return cls(config, metadata, incremental_config)
 
+    def get_table_description(  # pylint: disable=arguments-differ
+        self, schema_name: str, table_name: str, inspector: Inspector
+    ) -> Optional[str]:
+        description = None
+        try:
+            table_info: dict = inspector.get_table_comment(table_name, schema_name)
+            result = self.connection.execute(
+                sql.text(REDSHIFT_EXTERNAL_TABLE_LOCATION),
+                {"schema": schema_name, "table": table_name},
+            ).fetchone()
+            location = result[0] if result else None
+            if location:
+                self.external_location_map[
+                    (self.context.get().database, schema_name, table_name)
+                ] = location
+
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(
+                f"Table description error for table [{schema_name}.{table_name}]: {exc}"
+            )
+        else:
+            description = table_info.get("text")
+        return description
+
```

Comment:
this does not make sense to do it in `get_table_description` method, in other connectors we populate this map because while fetching the description we get the location details in a single query but here that is not the case 


see how we do it in snowflake https://github.com/open-metadata/OpenMetadata/blob/28bd01c471844d00183a97d67443e02e54e2be2b/ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py#L254

in a single query we extract the location for all the tables within a given schema this eliminates the need of running an extra query for each table